tools/torch2onnx.py

```diff
@@ -0,0 +1,126 @@
+import argparse
+import os.path as osp
+import sys
+import warnings
+
+import mmcv
+import torch
+import torch.nn as nn
+from mmcv.runner import load_checkpoint
+
+from mmaction.models import build_model
+
+try:
+    import onnx
+except ImportError:
+    warnings.warn('Please install onnx to support onnx exporting.')
+
+sys.path.append('../')
```

Comment:
can this line be smarter?

Reply:
How about `sys.path.append(osp.dirname(__file__))`

Reply:
Or just run by `python tools/torch2onnx.py xxx` at the `$MMAction2` path

Reply:
the latter is better

Reply:
Aggree. It is removed.